Source/Core/Core/PowerPC/Interpreter/Interpreter_Tables.cpp

```diff
@@ -88,10 +90,10 @@ static GekkoOPTemplate primarytable[] =
 	{61, Interpreter::psq_stu,      {"psq_stu", OPTYPE_STOREPS, FL_IN_FLOAT_S | FL_OUT_A | FL_IN_A | FL_USE_FPU | FL_LOADSTORE, 1, 0, 0, 0}},
 
 	//missing: 0, 1, 2, 5, 6, 9, 22, 30, 62, 58
-};
+}};
 
-static GekkoOPTemplate table4[] =
-{    //SUBOP10
+static std::array<GekkoOPTemplate, 13> table4 =
```

Comment:
> Maybe not the right place to talk about it, but is duplicating the size information avoidable?

As in being present in both the extern and definition? If that's the case, then yeah. Size is bound to the type signature, so it needs to be present.

> Or does it compile-error if a row is deleted or added without updating the number?

A compile-error will happen if you try to add a row past the size indicated.

Once the remaining parts of the [Library Fundamentals TS v2](http://en.cppreference.com/w/cpp/experimental/lib_extensions_2) is accepted for standardization, [std::make_array](http://en.cppreference.com/w/cpp/experimental/make_array) will make creating std::arrays less of a pain, as you could just do:

```cpp
auto some_table = std::make_array<type>(<elem>, <elem>, <elem>, ...);
```
and it'll automatically create an array with the signature `std::array<type, N>` where N is the number of elements within the parentheses.

We could just borrow the example implementation off of the page for the time being if that's more appropriate.